app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_create :update_post_comments_counter
+
+  def update_post_comments_counter
+    post.update(comments_counter: post.comments.count)
+  end
+end
```

Comment:
[OPTIONAL] The rails call-back method `after_create` registers a callback to be called around the creation of a record whilst the `after_save` method registers a callback to be called around the save of a record during creation or updating. Hence it will be prudent to use the `after_save` method here instead. 
Kindly also consider making the `update_post_comments_counter` method private since it will be called by the rails active callback method. :wink:
These recommendations apply to both the like and post model also. :+1: